src/Uno.UWP/Phone/Devices/Notification/VibrationDevice.Android.cs

```diff
@@ -0,0 +1,58 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Support.V4.Content;
+
+namespace Windows.Phone.Devices.Notification
+{
+	public partial class VibrationDevice
+	{
+		private const string Permission = "android.permission.VIBRATE";
+		
+		private readonly Vibrator _vibrator;
+
+		private static VibrationDevice _instance;
+
+		private VibrationDevice(Vibrator vibrator) =>
+			_vibrator = vibrator;
+
+		public static VibrationDevice GetDefault()
+		{
+			if (_instance == null)
+			{
+				if (ContextCompat.CheckSelfPermission(Application.Context, Permission) == Android.Content.PM.Permission.Denied)
+				{
+					throw new InvalidOperationException($"You need to declare {Permission} in AndroidManifest.xml");
+				}
+				var vibrator = Application.Context.GetSystemService(Context.VibratorService) as Vibrator;
```

Comment:
I understand now why UWP named the class `VibrationDevice`

Reply:
😂😂😂